packages/blueprints/gen-ai-chatbot/src/blueprint.ts

```diff
@@ -219,11 +233,28 @@ export class Blueprint extends ParentBlueprint {
     // write a workflow to my repository
     new Workflow(this, repository, workflowBuilder.getDefinition());
 
+    const claudeBehavior = this.claudeVersionCheck();
+    if (claudeBehavior?.from2To3) {
```

Comment:
Do we actually even need this? Why not just increment the blueprint version?

Reply:
sidenote: It would be nice to be able to autogenerate this from release notes or something similar. We shpuld think about this (obviously outside this PR)